great-idea/css/index.css

```diff
@@ -63,4 +63,160 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+@keyframes header-anima {
+    0%   {opacity:1;}
+    25%  {opacity:0.25;}
+    50%  {opacity:1;}
+    75%  {opacity:0.25;}
+    100% {opacity:1;}
+}
+
+/* General Styles */
+* {
+	font-family: 'Titillium Web', sans-serif;
+}
+
+p {
+	font-size: 16px;
+	line-height: 1.25;
+}
+
+.container {
+	/* border: 1px solid black; */
+	width: 880px;
+	margin: 0 auto;
+}
+
+.container header {
+	position: sticky;
+	top: 0;
+	padding: 30px 5px 10px 5px;
+	display: flex;
+	justify-content: space-between;
+	margin-bottom: 30px;
+	background-color: #F4F3F4;
+}
+
+.container header nav {
+	color: #ACB1B6;
+	width: 70%;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.container header nav a {
+	text-decoration: none;
+	color: inherit;
+}
+
+.container header nav a:hover {
+	animation-name: header-anima;
+	animation-duration: 4s;
+	animation-iteration-count: infinite;
+}
+
+.container footer {
+	/* border: 1px solid red; */
+	text-align: center;
+	margin: 100px 0 20px 0;
+}
+
+.container section {
+	margin-bottom: 30px;
+}
+
+.main-content {
+	width: 100%;
+	border-top: 1.5px solid black;
+	border-bottom: 1.5px solid black;
+}
+
+/* Home Styles */
+.index .cta {
+	display: flex;
+	justify-content: space-between;
+}
+
+.index .cta-text {
+	width: 60%;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	/* border: 1px solid red; */
+}
+
+.index h1 {
+	font-size: 4em;
+	text-align: center;
+}
+
+.index .top-content {
+	padding: 30px 0;
+	display: flex;
+	justify-content: space-between;
+}
+
+.index .bottom-content {
+	padding: 30px 0;
+	display: flex;
+	justify-content: space-between;
+}
+
+.index .text-content {
+	padding-left: 10px;
+}
+
+.index .contact p {
+	margin: 20px 0px;
+}
+
+
+/* Services Styles */
```

Comment:
I know we spoke about this.  Just remember to either split up the css into different files or have the organized into one.